packages/crud/src/vaadin-crud.js

```diff
@@ -1133,7 +1133,7 @@ class Crud extends ControllerMixin(ElementMixin(ThemableMixin(PolymerElement)))
     if (!item) {
       setTimeout(() => {
         this.__keepOpened = keepOpened;
-        this.editedItem = this._grid.activeItem = undefined;
+        this.editedItem = this._grid.undefined;
```

Comment:
This change looks odd (`this._grid.undefined`)

Reply:
My bad, fixed 🙈